docs_src/security/Ch-DelayedStartServices.md

```diff
@@ -0,0 +1,119 @@
+# Delayed-Start Services
+
+In some use cases, it is not possible to deliver a secret store token
+to an EdgeX microservice at the time the framework is started.
+This may be because a service is optional,
+because it is transient (doesn't run all the time),
+or because it may be difficult to deliver the token generated by security-secretstore-setup.
+
+To accommodate this use case,
+non-core EdgeX microservices have an ability to obtain their secretstore tokens
+via SPIFFE workload attestation.
+Non-core EdgeX microservices have SPIFFE support compiled into their binaries by default,
+and core services are compiled with a `non_delayedstart` build flag which removes this functionality for space reasons.
```

Comment:
it is possible to also make the core services with delayedstart, so probably could mention that for reference : [see here](https://github.com/edgexfoundry/edgex-go#delayed-start-go-builds-for-developers)

Reply:
Added note to that effect.